app/src/main/java/com/hover/stax/bounties/BountyAsyncCaller.java

```diff
@@ -14,18 +13,21 @@
 
 import java.io.IOException;
 import java.lang.ref.WeakReference;
+import java.util.HashMap;
+import java.util.Map;
 
 import okhttp3.MediaType;
 import okhttp3.OkHttpClient;
 import okhttp3.Request;
 import okhttp3.RequestBody;
 import okhttp3.Response;
+import timber.log.Timber;
 
-class BountyAsyncCaller extends AsyncTask<String, Void, Integer> {
+class BountyAsyncCaller extends AsyncTask<String, Void, Map<Integer, String>> {
```

Comment:
This should be replaced with modern implementation

Reply:
Yes. I'm gonna add this to Pivotal.